.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.png filter=lfs diff=lfs merge=lfs -text
+*.mp3 filter=lfs diff=lfs merge=lfs -text
+*.wav filter=lfs diff=lfs merge=lfs -text
```

Comment:
FYI this is the file that indicates what files should be checked in using LFS or using normal git. You can modify this file by hand or with the CLI like this: `git lfs track "*.psd"` for example. Right now only the 3 file types mentioned are being tracked. If any other file types are added we should make sure they get tracked here.